3기/LeeGaYeon/week1/page1/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" type="text/css" href="style.css" />
```

Comment:
```suggestion
    <link rel="stylesheet" href="style.css" />
```

HTML5 부터는 type의 생략이 가능해요. 😊

Reply:
미션2 figma url입니다.
https://www.figma.com/file/qq9sjaOxv2UUCzR9UyDFde/mission-2?node-id=2%3A122&t=ul1z7kST3vL3iZcI-1

이렇게 보내드리면 될까요?

Reply:
두번째 미션에 대해서 [피그마](https://www.figma.com/file/g2ajqMmtSzZhUFQVg5lYgU/3th-week1?node-id=1223%3A272&t=kRaMrBxa2Rqx5ftY-1)로 피드백 드렸습니다!

1주차 라이브세션때 사용한 피그마 시안 URL이예요. 😊